scripts/build/companion_libs/050-zlib.sh

```diff
@@ -94,20 +94,20 @@ do_zlib_backend() {
         "${extra_config[@]}"
 
     CT_DoLog EXTRA "Building zlib"
-    CT_DoExecLog ALL make ${JOBSFLAGS}
+    CT_DoExecLog ALL ${make} ${JOBSFLAGS}
```

Comment:
Ah you're correct. For at least one of my builds it used /usr/bin/make. Maybe something messed up there.